crates/fuel-core/Cargo.toml

```diff
@@ -39,7 +39,7 @@ fuel-core-types = { workspace = true, features = [
 ] }
 futures = { workspace = true }
 hex = { version = "0.4", features = ["serde"] }
-hyper = { version = "0.14" }
+hyper = { version = "0.14.26" }
```

Comment:
You can move it on workspace level and reuse in all places `{ workspace = true }`